Core/Frameworks/BaikalAdmin/Resources/main.js

```diff
@@ -24,4 +24,4 @@ function copyToClipboard(el) {
     sel.removeAllRanges();
     $(el).css({backgroundColor:"#75c753"});
     $(el).animate({backgroundColor:"transparent"}, 1500);
-}
+}
```

Comment:
Please revert the changes to the two files that are now no longer relevant for your PR